src/middlewares/Conflict.js

```diff
@@ -44,6 +46,26 @@ class Conflict {
   static isTripRequestFound(req, res, next) {
     return DataEngine.findOne(req, res, next, tripRequests, { id: req.params.tripRequestId }, 'The trip request id not found');
   }
+
+  /**
+    * Check if accomodation already exist
+    * @static
+    * @param {object} req request object
+    * @param {object} res response object
+    * @param {object} next next
+    * @memberof Conflict
+    * @returns {object} data
+    */
+  static checkForDuplicateAccommodation(req, res, next) {
+    return DataEngine.findConflict(req, res, next, accommodations,
+      {
+        name: req.body.name,
+        cityId: req.body.cityId,
+        address: req.body.address,
+        description: req.body.description,
+        googleCoordinates: req.body.googleCoordinates
```

Comment:
Would mind destructuring accommodation details?
It will look like:
```
const { name, cityId, address, description, googleCoordinates } = req.body;
return DataEngine.findConflict(req, res, next, accommodations,
      {
        name,
        cityId,
        address,
        description,
        googleCoordinates
```
I think this will look cleaner and organized.